.github/workflows/ci.yml

```diff
@@ -120,7 +119,6 @@ jobs:
     secrets: inherit
     needs:
       - deploy-flutter-cluster
-      - create-flutter-shared-apps
```

Comment:
why did we remove these steps. Don't we need it here. If not it is not obvious why?

Reply:
Creating apps may fails at the middle of creating. We have to clean at end in any case. Otherwise, `cleanup-flutter-shared-apps` doesn't happen if  `create-flutter-shared-apps` has failed.